### DISCUSSION TITLE
Using `rpi` Options Make the Experience Worse

### DISCUSSION THREAD

**Discussion by monomycelium** [FDH2/UxPlay#108] [Q&A]
I have rebuilt UxPlay for the [`Video4Linux2` plugin patches](https://github.com/FDH2/UxPlay/wiki/Gstreamer-Video4Linux2-plugin-patches). When I used the `rpi` options on my Raspberry Pi 4 (`uxplay -rpifb` when LightDM was turned off and `uxplay -rpi` when `PIXEL` was running), the screen mirroring seemed laggier than when I did not use the options. This is fine with me since not using the `rpi` options already gives me descent performance, except that I cannot have the screen mirroring window in fullscreen mode, which, as much as I know, can only be done using the `rpifb` option. Should I have installed other plugins for the `Video4Linux2` plugin?

**Comment by fduncanh** [comment]
see issue #109
